src/managers/retention-policies.js

```diff
@@ -322,4 +325,56 @@ RetentionPolicies.prototype.getAllFileVersionRetentions = function(options, call
 	return this.client.wrapWithDefaultHandler(this.client.get)(apiPath, params, callback);
 };
 
+/**
+ * Get files under retention by each assignment
+ * To use this feature, you must have the manage retention policies scope enabled
+ * for your API key via your application management console.
+ *
+ * API Endpoint: '/retention_policy_assignments/:assignmentID/files_under_retention'
+ * Method: GET
+ *
+ * @param {string} assignmentID - The Box ID of the policy assignment object to fetch
+ * @param {Object} [options] - Additional options for the request. Can be left null in most cases.
+ * @param {Array} [options.fields] - An array of fields to return: disposition_at, winning_retention_policy
```

Comment:
`fields` is usually defined as `{string} [options.fields] - Comma-separated list of fields to include in the response: disposition_at, winning_retention_policy`